docs/_documentations/codewind-openapi-vscode.md

```diff
@@ -22,6 +22,21 @@ The Codewind OpenAPI Tools for VS Code provides commands that invoke the OpenAPI
 3. This extension pulls the [OpenAPI Generator CLI Docker Image](https://github.com/OpenAPITools/openapi-generator#16---docker) and runs the OpenAPI Generator in a Docker container. Install Docker if necessary.
 4. Install the Codewind OpenAPI Tools from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-openapi-tools) or by searching for `Codewind OpenAPI` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
 
+## Generating and building code in an existing Java Maven project
+1. Right-click your Codewind project and launch the **Generate Server from OpenAPI Definition** wizard.
+2. Select your OpenAPI document `.json` file.
+3. Select the `jaxrs-spec` server generator type.
+4. When you're prompted to select the output folder, keep the default selection. The project root, which is the parent of the source folder, is selected by default.
+5. Click **Select As Output Folder** to merge the generated `.pom` file with the existing `.pom` file. The merge attempts to include all necessary dependencies for the project to build properly.
+6. When prompted with the warning message, note that the `pom.xml` file is backed up. However, XML comments, such as `<!-- example comment -->`, are removed. Click **Yes** to proceed. The files build.
+7. See that your `src` folder shows the newly generated code complete with a `gen` package. Your project state returns to the `[Running]` state. The `src/main/java` and `src/test/java` are not customizable.
```

Comment:
7. See that your `src` folder shows the newly generated code. Your Codewind project state returns to the `[Running]` state. The target source folders `src/main/java` and `src/test/java` are currently not customizable.

(Removed: `complete with a `gen` package` because the gen folder is specific to the generator that is chosen)